### PR TITLE
Updated LanceDB Doc

### PR DESCRIPTION
**Pull request by PrashantDixit0**
## Description

Please include a summary of the change and which issue is fixed. Please also include relevant motivation and context. List any dependencies that are required for this change.

Fixes # (issue)

## Type of change

Please delete options that are not relevant.

- [ ] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [ ] Refactor (does not change functionality, e.g. code style improvements, linting)
- [x] Documentation update

## How Has This Been Tested?

Please describe the tests that you ran to verify your changes. Provide instructions so we can reproduce. Please also list any relevant details for your test configuration

Please delete options that are not relevant.

- [ ] Unit Test
- [ ] Test Script (please provide)

## Checklist:

- [x] My code follows the style guidelines of this project
- [x] I have performed a self-review of my own code
- [x] I have commented my code, particularly in hard-to-understand areas
- [x] I have made corresponding changes to the documentation
- [x] My changes generate no new warnings
- [ ] I have added tests that prove my fix is effective or that my feature works
- [ ] New and existing unit tests pass locally with my changes
- [ ] Any dependent changes have been merged and published in downstream modules
- [x] I have checked my code and corrected any misspellings

## Maintainer Checklist

- [ ] closes #xxxx (Replace xxxx with the GitHub issue number)
- [ ] Made sure Checks passed

### CODE REVIEW

**Comment by PrashantDixit0** [comment]
@Dev-Khant Here I have updated Documentation of LanceDB, Please review it

File: docs/components/vector-databases/lancedb.mdx [hunk 1]
```diff
@@ -45,4 +47,46 @@ while(True):
...
+  }
+}
+
+app = App.from_config(config=config)
+
+# add data source and start query in
+app.add("https://www.tesla.com/ns_videos/2022-tesla-impact-report.pdf")
+
+# query
+app.chat("What are Tesla's key sustainability achievements in 2022?")
```
Comment (Dev-Khant) [inline docs/components/vector-databases/lancedb.mdx:89]: Please also add the response of the chat.

File: docs/components/vector-databases/lancedb.mdx [hunk 2]
```diff
@@ -45,4 +47,46 @@ while(True):
 ```
 
 </CodeGroup>
-<Snippet file="missing-vector-db-tip.mdx" />
```
Comment (Dev-Khant) [inline docs/components/vector-databases/lancedb.mdx:48]: Also include this line at the end of the doc

Comment (PrashantDixit0) [reply docs/components/vector-databases/lancedb.mdx:48]: Done :+1:

**Review by deshraj — APPROVED** [review]

**Pull request merged**